src/Uno.UWPSyncGenerator/DocGenerator.cs

```diff
@@ -131,8 +131,8 @@ void BuildMemberLists()
 							{
 								_sb.AppendParagraph($"*Implemented for:* {ToDisplayString(view.ImplementedForMain)}");
 
-								var baseDocLinkUrl = @"https://docs.microsoft.com/en-us/uwp/api/" + view.UAPSymbol.ToDisplayString().ToLowerInvariant();
-								_sb.AppendParagraph($"This document lists all properties, methods, and events of {formattedViewName} that are currently implemented by the Uno Platform. See the {Hyperlink("WinUI and UWP documentation", baseDocLinkUrl)} for detailed usage guidelines which all automatically apply to Uno Platform. ");
+								var baseDocLinkUrl = @"https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/" + view.UAPSymbol.ToDisplayString().ToLowerInvariant();
```

Comment:
I'm not sure if **every single API** will have the same url format. @jeromelaban Have you got an idea?

Reply:
I've tested around, and it seems to be working for all properties that I've tested in the output of this PR, so we can keep it this way, thanks!